### DISCUSSION TITLE
Error while trying to load BERTopic model in Sagemaker end point with PyTorch

### DISCUSSION THREAD

**Discussion by AshokNagula** [MaartenGr/BERTopic#1163] [Q&A] [answered]
I am trying to make a Sage maker pipeline, successfully trained the model and created end point with PyTorch framework.
when i try to make inference from the model , i'm facing the following error which seems like internal package.
using bertopic -0.14 on both training and inference instances , python - 3.9.

any thoughts  @MaartenGr ?

<img width="1664" alt="Screenshot 2023-04-06 at 10 06 33 AM" src="https://user-images.githubusercontent.com/38754565/230275185-7d3b5112-150a-428e-9ca6-35c60f3305fd.png">

**Comment by MaartenGr** [comment, marked as answer]
I believe there are a [number of closed issues](https://github.com/MaartenGr/BERTopic/issues?q=is%3Aissue+correct_alternative_cosine+is%3Aclosed) mentioning this specific issue. Did you already try out the suggestions there? From what I remember, it is due to some dependency problems with numpy.

**Reply by AshokNagula** [reply]
yes , Its resolved with Numpy==1.23.3 ,PyTorch == 1.10 and Python==3.8. Thanks !
